README.md

```diff
@@ -1,2 +1,40 @@
-# nlp-in-10-lines
-Repository for PyCons 2016 workshop Natural Language Processing in 10 Lines of Code.
+### PyCon UK 2016 
+![NLP in 10 Lines of Code](http://i.imgur.com/TuusTAm.png)
+# Natural Language Processing in 10 Lines of Code
+##### from Andraz Hribernik, Matic Horvat, Aeneas Wiener, Tom Sherborne
+
+At [Cytora](www.cytora.com) we use NLP to extract and analyse plain text to build our structured information product.
```

Comment:
Link does not work. Maybe you need to change `www.cytora.com` to `http://www.cytora.com`